platform-configs/include/novacustom-mtl.robot

```diff
@@ -0,0 +1,30 @@
+*** Variables ***
+# Flash
+${FLASH_SIZE}=                      ${16*1024*1024}
+
+# CPU
+${INITIAL_CPU_FREQUENCY}=           2800
+${DEF_CORES_PER_SOCKET}=            4
+${DEF_THREADS_PER_CORE}=            2
+${DEF_THREADS_TOTAL}=               ${EMPTY}
+# TODO: remove, the value below can be inferred from the one above
+${DEF_ONLINE_CPU}=                  0-7
+${DEF_SOCKETS}=                     1
+
+# Audio
+${DEVICE_AUDIO1}=                   ALC245
+${DEVICE_AUDIO2}=                   Tigerlake HDMI
```

Comment:
Should be something like `Meteor Lake-P HDMI`. You can obtain it with `cat /proc/asound/card0/codec\#2 | grep -i "Codec:"`

Reply:
fixed in: ebada758dbf9ba9229c9a44163d599ee96603f52